repository_service_tuf_api/metadata.py

```diff
@@ -85,6 +85,63 @@ def post_metadata(payload: MetadataPostPayload) -> MetadataPostResponse:
     return MetadataPostResponse(data=data, message=message)
 
 
+class MetadataPutPayload(BaseModel):
+    roles: List[Roles.values()]
+
+    class Config:
+        example = {"roles": ["targets", "snapshot"]}
+
+        schema_extra = {"example": example}
+
+
+class MetadataPutResponse(BaseModel):
+    data: Optional[TaskData]
+    message: str
+
+    class Config:
+        example = {
+            "data": {
+                "task_id": "7a634b556f784ae88785d36425f9a218",
+                "last_update": "2022-12-01T12:10:00.578086",
+            },
+            "message": "Force new online metadata accepted.",
+        }
+
+        schema_extra = {"example": example}
+
+
+def put_metadata(payload: MetadataPutPayload) -> MetadataPutResponse:
+    bs_state = bootstrap_state()
+    if bs_state.bootstrap is False:
+        raise HTTPException(
+            status.HTTP_200_OK,
+            detail={
+                "message": "Task not accepted.",
+                "error": (
+                    f"Requires bootstrap finished. State: {bs_state.state}"
+                ),
+            },
+        )
+
```

Comment:
We should verify that it is an online role. See my comment on https://github.com/repository-service-tuf/repository-service-tuf-worker/pull/435 for more detail.